src/backend/templates/schedule.hbs

```diff
@@ -514,6 +514,9 @@
 
           $('.date-button').click(function() {
               let className = $(this).attr('class').split(' ')[0];
+              $(this).css({'background-color': '#EEE', 'color': 'black'});
```

Comment:
@Princu7 
Instead of doing this, Add a class to the link which is active.
This will give us a consistent way to use active class in all pages, whenever a button is clicked.